test/functional-tests/index.js

```diff
@@ -5,6 +5,14 @@ const Browser = require('./lib/browser');
 const App = require('./lib/app');
 const assert = require('assert');
 
+// TO BE REVISITED FOR CREATING MOCK API CALL
```

Comment:
Would also be good to write some unit tests after you refactor the index.js of the component with more functions, test them individually